### PR TITLE
Drop unused AccessControlEntry table

### PR DESCRIPTION
**Pull request by hsong-rh**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -78,6 +65,7 @@
     t.string "owner"
     t.bigint "parent_id"
     t.bigint "request_context_id"
+    t.string "random_access_key"
```
Comment (bzwei) [inline db/schema.rb:68]: should not have this change

File: db/schema.rb [hunk 2]
```diff
@@ -86,6 +74,7 @@
     t.datetime "finished_at"
     t.index ["group_ref"], name: "index_requests_on_group_ref"
     t.index ["parent_id"], name: "index_requests_on_parent_id"
+    t.index ["random_access_key"], name: "index_requests_on_random_access_key"
```
Comment (bzwei) [inline db/schema.rb:77]: should not have it

**Comment by gmcculloug** [comment]
@eclarizio Please review

File: db/schema.rb (hunk 2 above)
Comment (hsong-rh) [reply db/schema.rb]: Removed these auto generated changes

**Pull request merged**